filament/src/Froxelizer.cpp

```diff
@@ -104,12 +104,12 @@ Froxelizer::Froxelizer(FEngine& engine)
     static_assert(std::is_same_v<RecordBufferType, uint8_t>,
             "Record Buffer must use bytes");
 
-    if (UTILS_UNLIKELY(engine.getActiveFeatureLevel() == FeatureLevel::FEATURE_LEVEL_0)) {
+    DriverApi& driverApi = engine.getDriverApi();
+
+    if (UTILS_UNLIKELY(driverApi.getFeatureLevel() == FeatureLevel::FEATURE_LEVEL_0)) {
```

Comment:
Yes! Do we agree that we have an invariant that driver feature level 0 *implies* engine feature level 0, but the reverse is no longer true (it was before because it wasn't possible to lower the feature level, which started at least at 1 on non ES2 devices).

Reply:
Yes, this seems to me like the correct way to think about the state of the engine now.